Mathlib/Algebra/Order/Interval/Set/SuccPred.lean

```diff
@@ -67,13 +67,24 @@ lemma insert_Icc_eq_Icc_add_one_right (h : a ≤ b + 1) :
     insert (b + 1) (Icc a b) = Icc a (b + 1) := by
```

Comment:
`insert_Icc_eq_Icc_add_one_right` looks like you want to change it too?

Reply:
(In general, the rule you told me in person was that the left/right indicator, if it exists, should be associated to the LHS - that rule seems reasonable and I think you're following it in the new lemmas!)